### DISCUSSION TITLE
'useSignInCheck' is not exported from 'reactfire'

### DISCUSSION THREAD

**Discussion by attilioweb** [FirebaseExtended/reactfire#385] [Q&A]
I keep receiving the following error during build time `'useSignInCheck' is not exported from 'reactfire'`

I originally had done `npm install reactfire` but after receiving that error and doing some Google searches I came across this page https://www.gitmemory.com/issue/FirebaseExtended/reactfire/375/849969933 but `npm i reactfire@canary` also produces the error. Deleting node_modules and reinstalling didn't help either.

Any thoughts?

**Comment by conrado** [comment]
try `useSigninCheck` instead, there was a typo in the docs that has already been fixed
